doc/about.rst

```diff
@@ -501,6 +499,9 @@ the past:
 .. |aphp| image:: images/logo_APHP_text.png
   :target: https://aphp.fr/
 
+.. |hf| image:: images/huggingface_logo-noborder.png
```

Comment:
you might want to remove the image file as well in this PR.

Reply:
No need to remove it as the logo will still be displayed among the organizations that funded the consortium in the past.

Reply:
But thanks! I just noticed it is not being correctly rendered.

Reply:
Indeed. I forgot that we have this section.